environment/cpu/src/cloud/benchflow/monitors/cpu.go

```diff
@@ -245,7 +246,7 @@ func startMonitoring(w http.ResponseWriter, r *http.Request) {
 		ID := containerInspect.ID
 		statsChannel := make(chan *docker.Stats)
 		doneChannel := make(chan bool)
-		c := Container{ID: ID, statsChannel: statsChannel, doneChannel: doneChannel}
+		c := Container{Name: each, ID: ID, statsChannel: statsChannel, doneChannel: doneChannel}
```

Comment:
`each` is not equal to `Name`. Set the actual value in a variable as you do for the `ID`, and then set it here.